### DISCUSSION TITLE
Librechat article on Medium.com

### DISCUSSION THREAD

**Discussion by dirkpetersen** [danny-avila/LibreChat#4635] [General]
As my enterprise chat deployment project with LibreChat and AWS bedrock is in the final stages, I wrote an article about about the business case on Medium: https://dirk-petersen.medium.com/are-chatgpt-costs-killing-you-23646fe4bb94 . If you see any errors in there I'd appreciate a hint

**Comment by danny-avila** [comment]
Thanks will check it out!

**Comment by larryb821** [comment]
great article Dirk. i came to the same conclusion for my small firm. i wanted users to have access but since there usage is expected to be low this was a much more cost effective solution while we all figure out how to use AI as a team.
